external/@wordpress/elements/index.js

```diff
@@ -10,7 +16,7 @@
  * @return {wp.Element}                  Element
  */
 export function createElement( type, props, ...children ) {
```

Comment:
Do you think it's better to have a clear abstraction layer, or should we just export them straight from React?

```js
export { createElement } from 'react';
export { render } from 'react-dom';
```